### ISSUE TITLE
Retrieving OAuth2 credential returns empty string

### ISSUE THREAD

**Issue by dirkmoors** [Mashape/kong#536]
When retrieving an OAuth2 credential, I receive an empty string as the response (status code is 200, so that's ok)

I'm performing a GET request on the following endpoint:
```
http://localhost:8001/consumers/af94a77d-9dfa-4842-c8f5-7a1da7948a04/oauth2/5c75e28c-d00e-400e-c053-8269ab51658d/
```
Previously, I received the correct credential structure.

Am I doing something wrong or is this now expected behaviour?

**Comment by thefosk** [comment]
@dirkmoors I have just tested this with `master` and seems to be working properly. It may have been a bug in 0.4.2 that will be fixed in 0.5.0. Next week we are releasing 0.5.0 so you can try with the new version.

**Comment by thefosk** [comment]
@dirkmoors 0.5.0 has been released, please let me know if you still experience the issue.
